src/authenticated-app.tsx

```diff
@@ -2,13 +2,19 @@ import * as React from 'react'
 import { Switch, Route } from 'react-router-dom'
 
 import { ProjectTypeSelection } from 'screens/project-type-selection'
-import { SmallBuildQuoteCalculator } from 'screens/small-build-quote-calculator'
+// import { SmallBuildQuoteCalculator } from 'screens/small-build-quote-calculator'
+import { SmallBuildMaterials } from 'screens/small-build-materials'
+import { SmallBuildWages } from 'screens/small-build-wages'
+import { SmallBuildRentals } from 'screens/small-build-rentals'
 
 const AuthenticatedApp: React.FC = () => {
   return (
     <Switch>
       <Route exact path="/" component={ProjectTypeSelection} />
-      <Route path="/small-build" component={SmallBuildQuoteCalculator} />
+      {/* <Route path="/small-build" exact component={SmallBuildQuoteCalculator} /> */}
+      <Route path="/small-build/materials" component={SmallBuildMaterials} />
+      <Route path="/small-build/wages" component={SmallBuildWages} />
+      <Route path="/small-build/rentals" component={SmallBuildRentals} />
```

Comment:
I don't think we want to call these screens `small-build` they should apply to all the builds.

Reply:
Okay, cool.  I'll change that.